### DISCUSSION TITLE
[Art competition] The Undecidability of Mathematics

### DISCUSSION THREAD

**Discussion by pingpingy1** [prosyslab-classroom/cs424-program-reasoning#256] [Art competition]
![DALL·E 2022-11-30 20 33 10 - a taller tower of babel hidden in fog and the empire state building in construction, digital art](https://user-images.githubusercontent.com/94441510/204786670-ce9880b6-f1ad-4196-afef-3a9ce3b0fd88.png)

## Short Description
For me, [the undecidability of mathematics] means "[a taller tower of babel hidden in fog and the empire state building in construction, digital art]".

## Long Description (up to 3 sentences)
We once hoped to meet God himself by building a really high tower that reached Him, but we were damned to fail that project.
Likewise, we once hoped that all mathematics could be automated by building a sophisticated computer that will prove all true theorems, but we learned that such a task was impossible, proven by Turing and Rice.
Nonetheless, we still try to get as close as possible to the holy grail of perfect program synthesis, knowing we will fail, but successfully placing one more brick on the Empire State Building, making it that much taller and reaching that much closer to the tower of Babel.

**Comment by KihongHeo** [comment]
Good. Keep going.
